tests/unit/TodoList1.spec.ts

```diff
@@ -1849,7 +1850,7 @@ describe('TodoList - UI - Amalgamation Applications', () => {
 describe('TodoList - Click Tests - Coops', () => {
   const { assign } = window.location
 
-  beforeAll(() => {
+  beforeEach(() => {
```

Comment:
Does this really have to be set before each test?

Reply:
Yup, some of values were missing, that's why the tests were failing.